Tools/Build-SdkChangelog.ps1

```diff
@@ -0,0 +1,88 @@
+[CmdletBinding()]
+param (
+    [Parameter(Mandatory = $true, ValueFromPipelineByPropertyName = $true, HelpMessage = 'Name of the API to build an SDK for.')][ValidateSet('JumpCloud.SDK.DirectoryInsights', 'JumpCloud.SDK.V1', 'JumpCloud.SDK.V2')][ValidateNotNullOrEmpty()][System.String[]]$SDKName
+)
+$SdkChangelogFilePath = "./$SDKName.md"
+Import-Module ($PSScriptRoot + '/New-SdkChangelog.ps1')
+# For authorized requests add header & OAuth Token:
+# $headers = @{
+#     "Authorization" = "<OAuth Token>"
+# }
```

Comment:
Rate limit occurs when this triggers many times in one hour, we can add in Authorization via OAuth Tokens if this becomes an issue